extensions/amazon-dynamodb/deployment/src/main/java/io/quarkus/dynamodb/deployment/DynamodbProcessor.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.dynamodb.deployment;
+
+import javax.inject.Inject;
+
+import org.apache.commons.logging.impl.Jdk14Logger;
+import org.apache.commons.logging.impl.LogFactoryImpl;
```

Comment:
It would be better to use the JBoss Logging wrappers instead. Could you check if it's possible (basically, you would have to exclude the dependencies and use JBoss Logging dependencies instead, we have one for commons-logging and one for the JDK logging if need be)?

Reply:
I wasn't able to find any existing extension that swtichers to jboss logging. Any extension that registers Jdk14Logger & LogFactoryImpl for reflections doesn't have any modifications to the dependencies.  Did your intention is to simply exclude commons logging dependency and include jboss one ?

Reply:
I configured DEBUG logging of the dynamodb clients and looks like the underlying logging from quarkus was used. Added application properties to enable it

Reply:
Yes, the idea is to exclude the dependencies and then you shouldn't need to register these classes for reflection. At least if it works correctly :).